<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>225</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>413</width>
    <height>225</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>413</width>
    <height>225</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/main/icons/icons/cstrike.png</normaloff>:/main/icons/icons/cstrike.png</iconset>
  </property>
  <widget class="QCheckBox" name="LimitDownload">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Limit download speed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="LimitUpload">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Limit upload speed</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="KeepSeeding">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Continue seeding after download.</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="downloadLimitVal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="uploadLimitVal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>21</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>KB/s</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>21</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>KB/s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applySettings">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
